add `--helm` flag to `find` command (#154)
FairwindsOps/nova: docs/usage.md

```
Flags:

>       --containers        Show old container image versions instead of helm chart versions. There will be no helm output if this flag is set.
>       --helm              Show old helm releases instead of helm chart versions. You can combine this flag with `--containers` to have both output in a single run.

Reviewer: ```suggestion
      --containers        Show old container image versions. There will be no helm output unless the --helm flag is set as well
      --helm              Show old helm releases. You can combine this flag with `--containers` to have both output in a single run.
```

k8s.gcr.io/kube-apiserver             v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-controller-manager    v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-proxy                 v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-scheduler             v1.22.9            true    v1.25.0    v1.25.0          v1.22.13  
```

You can print the output in `json` format

```
$ nova --format=json find --helm --containers | jq
{
  "helm": [
    {
      "release": "cert-manager",
      "chartName": "cert-manager",
      "namespace": "cert-manager",
      "description": "A Helm chart for cert-manager",
      "home": "https://github.com/cert-manager/cert-manager",
      "icon": "https://raw.githubusercontent.com/cert-manager/cert-manager/d53c0b9270f8cd90d908460d69502694e1838f5f/logo/logo-small.png",
      "Installed": { "version": "v1.9.1", "appVersion": "v1.9.1" },
      "Latest": { "version": "1.9.1", "appVersion": "v1.9.1" },
      "outdated": false,
      "deprecated": false,
      "helmVersion": "3",
      "overridden": false
    }
  ],
  "include_all": false,

>   "container": {

Author: @rbren  - I merged `include_all` at the top level field and remove it from `containers` object. Any other concern/suggestion?

k8s.gcr.io/kube-apiserver             v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-controller-manager    v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-proxy                 v1.22.9            true    v1.25.0    v1.25.0          v1.22.13    
k8s.gcr.io/kube-scheduler             v1.22.9            true    v1.25.0    v1.25.0          v1.22.13  
```

You can print the output in `json` format

```
$ nova --format=json find --helm --containers | jq
{
  "helm": [
    {
      "release": "cert-manager",
      "chartName": "cert-manager",
      "namespace": "cert-manager",
      "description": "A Helm chart for cert-manager",
      "home": "https://github.com/cert-manager/cert-manager",
      "icon": "https://raw.githubusercontent.com/cert-manager/cert-manager/d53c0b9270f8cd90d908460d69502694e1838f5f/logo/logo-small.png",
      "Installed": { "version": "v1.9.1", "appVersion": "v1.9.1" },
      "Latest": { "version": "1.9.1", "appVersion": "v1.9.1" },
      "outdated": false,
      "deprecated": false,
      "helmVersion": "3",
      "overridden": false
    }
  ],
  "include_all": false,
  "container": {

>     "container_images": [

Reviewer: can we combine these? I see the `include_all` field is duplicated, but it's probably always set the same...

Author: I guess it is ok for `include_all` as it is the same flag... but `err_images` and `latest_string_found` lives in container context...
